templates/page--moderngov-page-tpl.html.twig

```diff
@@ -0,0 +1,99 @@
+{#
+/**
+ * @file
+ * Default ModernGov page template.
+ *
+ * Non-admin users see "{breadcrumb}", "{content}, and "{sidenav}" instead of
+ * the usual content.
+ *
+ * The doctype, html, head and body tags are not in this template. Instead they
+ * can be found in the html.html.twig template in this directory.
+ *
+ * Available variables:
+ *
+ * General utility variables:
+ * - base_path: The base URL path of the Drupal installation. Will usually be
+ *   "/" unless you have installed Drupal in a sub-directory.
+ * - is_front: A flag indicating if the current page is the front page.
+ * - logged_in: A flag indicating if the user is registered and signed in.
+ * - is_admin: A flag indicating if the user has permission to access
+ *   administration pages.
+ *
+ * Site identity:
+ * - front_page: The URL of the front page. Use this instead of base_path when
+ *   linking to the front page. This includes the language domain or prefix.
+ *
+ * Page content (in order of occurrence in the default page.html.twig):
+ * - messages: Status and error messages. Should be displayed prominently.
+ * - node: Fully loaded node, if there is an automatically-loaded node
+ *   associated with the page and the node ID is the second argument in the
+ *   page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - page.header: Items for the header region.
+ * - page.primary_menu: Items for the primary menu region.
+ * - page.secondary_menu: Items for the secondary menu region.
+ * - page.highlighted: Items for the highlighted content region.
+ * - page.help: Dynamic help text, mostly for admin pages.
+ * - page.content: The main content of the current page.
+ * - page.sidebar_first: Items for the first sidebar.
+ * - page.sidebar_second: Items for the second sidebar.
+ * - page.footer: Items for the footer region.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ *
+ * @ingroup themeable
+ */
+#}
+<div class="layout-container">
+
+  <header role="banner">
+    {{ page.header }}
+  </header>
+
+  {{ page.primary_menu }}
+  {{ page.secondary_menu }}
+
+  <div class="breadcrumb-wrapper">
+    {breadcrumb}
+  </div>
+
+  {{ page.highlighted }}
+
+  {{ page.help }}
+
+  {# Site admins should be able to edit/delete the ModernGov template page. #}
+  {% if is_admin and page.content_top %}
+    {{ page.content_top }}
+  {% endif %}
+
+  <main role="main">
+    <a id="main-content" tabindex="-1"></a>
+
+    <div class="layout-content">
+      {# Site admins should be able to edit/delete the ModernGov template page. #}
+      {% if is_admin %}
+        {{ page.content }}
+      {% else %}
+        {content}
+      {% endif %}
+    </div>
+
+    {% if page.sidebar_second %}
```

Comment:
I think we should remove the `if` around this and force the sidebar. If not, the moderngov sidebar will not be visible unless someone has placed a block in there. We can't be certain of this.

Reply:
Done.  I should have thought of this.